SmallBetterLogger/Source.cpp

```diff
@@ -0,0 +1,22 @@
+#include "SmallBetterLogger.hpp"
+
+int main()
```

Comment:
I don't think there should be a main here, but I believe it's for testing purposes for now.